carbonj.service/src/main/java/com/demandware/carbonj/service/engine/cfgCarbonJ.java

```diff
@@ -170,6 +170,9 @@
 
     @Value( "${app.servicedir:}" ) private String serviceDir;
 
+    @Value( "${kinesis.consumer.region:us-east-1}" )
+    private String kinesisConsumerRegion = "us-east-1";
```

Comment:
I don't think this needs to be explicitly assigned because the above line is setting a default of the same value.

Reply:
It will be overridden with the value specified in application.yaml